From 815698bfa8b92e5d6283e390eb5ed2a839127e8d Mon Sep 17 00:00:00 2001
From: Hugo Parente Lima <hugo.pl@gmail.com>
Date: Wed, 23 Sep 2015 00:47:10 -0300
Subject: [PATCH] Use MD5 for project configuration hash.

Ruby hash function uses some internal ruby data, so the same string
doesn't have the same hash across multiple interpreter runs.

diff --git a/app/models/project.rb b/app/models/project.rb
index 4ca8354..0dc1906 100644
--- a/app/models/project.rb
+++ b/app/models/project.rb
@@ -25,7 +25,7 @@ class Project < ActiveRecord::Base
   end
 
   def configuration_hash
-    linter.hash.to_s
+    Digest::MD5.hexdigest(linter)
   end
 
   private
-- 
2.7.3

